charts/postgres-operator/crds/postgresqls.yaml

```diff
@@ -185,8 +185,10 @@ spec:
               # Note: usernames specified here as database owners must be declared in the users key of the spec key.
             dockerImage:
               type: string
-            enableConnectionPooler:
+            enableMasterConnectionPooler:
```

Comment:
unfortunately I am not convinced that we can rename right now, so we should add the new master field and then it needs to be 'pooler || masterPooler' for backwards compatibility